cpp/harvest/nacjd.cc

```diff
@@ -119,6 +135,17 @@ void HandleChar(const char c) {
     }
 }
 
+std::string CheckingStatisticsCategory(std::string data_title) {
+    std::string category_name("");
+    for (std::string n : statistics_categories_array) {
```

Comment:
usually when iterating over a container and you do it like this, a copy of all variables is created. you can iterate by reference instead like this:

`for (const std::string &n : statistics_categories_array) {`